css/styles.css

```
@@ -164,269 +191,670 @@ body {
 .header-container {
   display: flex;
   flex-direction: column;
-  margin-top: 45px;
+  margin-top: 40px;
   position: relative;
 }
+
 .header-logo-image {
   position: absolute;
   display: block;
   top: -25px;
   left: 2px;
 }
+
 /* Пользовательское меню - user-navigation-list */
 .user-navigation {
   display: flex;
-  margin-top: 34px;
-  margin-left: 45px;
-  margin-bottom: 14px;
+  align-items: center;
+  margin-top: 35px;
+  margin-left: 42px;
+  margin-bottom: 15px;
 }
+
 .user-navigation-link {
-  display: block;
+  display: flex;
+  height: 40px;
   font-size: 16px;
-  line-height: 23px;
+  line-height: 18px;
   padding: 0;
   margin: 0;
   border: 2px solid transparent;
   box-sizing: border-box;
   border-radius: 20px;
   outline: none;
 }
+
 .search-form {
   display: flex;
   align-items: center;
-  background: url(../img/icon/search.svg) no-repeat 16px center;
-  margin-right: 19px;
   position: relative;
+  width: 500px;
+}
+
+.search-icon {
+  position: absolute;
+  top: 0;
+  left: 16px;
+  bottom: 0;
+  margin: auto;
+  display: block;
 }
+
 .search-input {
-  min-width: 356px;
+  width: 500px;
+  height: 40px;
+  box-sizing: border-box;
   font-family: inherit;
   font-size: 16px;
   line-height: 19px;
-  color: #000000;
-  /* opacity: 0.3; */
-  background-color: transparent;
+  background: none;
+  opacity: 0.3;
+  flex-grow: 1;
+  position: relative;
+  padding: 10px 160px 10px 52px;
   border: 2px solid transparent;
-  padding: 0;
-  margin: 0;
   border-radius: 20px;
-  padding: 10px 20px 10px 52px;
-  box-sizing: border-box;
-  outline: none;
+  margin: 0;
 }
+
 .search-button {
-  min-width: 160px;
+  position: absolute;
+  height: 40px;
+  right: 0;
+  top: 0;
+  margin: auto;
+  box-sizing: border-box;
+  width: 160px;
   font-family: inherit;
   font-size: 16px;
   line-height: 19px;
-  text-transform: none;
-  letter-spacing: 0;
-  color: #000000;
-  background-color: transparent;
-  border: 2px solid #000000;
-  box-sizing: border-box;
-  border-radius: 0px 20px 20px 0px;
-  padding: 10px 47px;
+  border: none;
+  background: none;
+  cursor: pointer;
+  display: none;
+}
+
+.navigation-user {
+  display: flex;
+  flex-wrap: wrap;
+  width: 540px;
   margin: 0;
-  outline: none;
-  visibility: hidden;
+  padding: 0;
+  list-style-type: none;
+  margin-left: 21px;
+}
+
+.navigation-user-item {
+  position: relative;
 }
+
 .login-title {
   max-width: 117px;
-  padding: 11px 21px 10px 46px;
+  padding: 10px 21px 10px 41px;
   margin-right: 141px;
-  background: url(../img/icon/login.svg) no-repeat 21px center;
+  background: url(../img/icon/login.svg) no-repeat 16px center;
```

```suggestion
  background: url("../img/icon/login.svg") no-repeat 16px center;
```
на всех url пути лучше добавить кавычки